crates/bevy_render/src/view/window.rs

```diff
@@ -164,48 +164,53 @@ pub fn prepare_windows(
 ) {
     let window_surfaces = window_surfaces.deref_mut();
     for window in windows.windows.values_mut() {
-        let surface = window_surfaces
-            .surfaces
-            .entry(window.id)
-            .or_insert_with(|| unsafe {
-                // NOTE: On some OSes this MUST be called from the main thread.
-                render_instance.create_surface(&window.handle.get_handle())
-            });
-
-        let swap_chain_descriptor = wgpu::SurfaceConfiguration {
-            format: TextureFormat::bevy_default(),
-            width: window.physical_width,
-            height: window.physical_height,
-            usage: wgpu::TextureUsages::RENDER_ATTACHMENT,
-            present_mode: match window.present_mode {
-                PresentMode::Fifo => wgpu::PresentMode::Fifo,
-                PresentMode::Mailbox => wgpu::PresentMode::Mailbox,
-                PresentMode::Immediate => wgpu::PresentMode::Immediate,
-                PresentMode::AutoVsync => wgpu::PresentMode::AutoVsync,
-                PresentMode::AutoNoVsync => wgpu::PresentMode::AutoNoVsync,
-            },
-        };
-
-        // Do the initial surface configuration if it hasn't been configured yet. Or if size or
-        // present mode changed.
-        if window_surfaces.configured_windows.insert(window.id)
-            || window.size_changed
-            || window.present_mode_changed
-        {
-            render_device.configure_surface(surface, &swap_chain_descriptor);
+        if window.raw_window_handle.is_none() {
+            continue;
```

Comment:
This feels like it should just be an `else` branch on the if let Some below?

Reply:
Avoiding indentation.

Reply:
This `continue` doesn't seem to be doing anything?

Reply:
It skips the rest of the loop.

Reply:
IMO this needs to be clarified in some form; if it's confusing reviewers it'll confuse code readers later.

Reply:
The the rest of the loop is just one big if statement with the same condition. That makes the continue redundant.

Reply:
Is there a more succinct way to code this?  The `continue` isn't redundant because one can't call `as_ref()` on `None`, can one?

I agree a line documenting the reason to skip the rest of the loop should be added.

Reply:
Updated the code to be more clear and succinct.  Thanks, Rust Programming Language Community Server Discord!

Reply:
All changes complete on this PR.  Ready for final review.